libdnf/sack/query.cpp

```diff
@@ -710,6 +710,11 @@ Query::getResult() noexcept
 }
 
 const Map * Query::getResult() const noexcept { return pImpl->result->getMap(); }
+PackageSet * Query::getResultPset() noexcept
```

Comment:
Are you sure with `noexcept`? It calls `apply()`. The method `apply()` is not marked as `noexcept`. And its inner methods also are not marked as `noexcept`.

Reply:
Thanks